### PR TITLE
Add sccache s3 controls

### PR DESCRIPTION
**Pull request by robertmaynard**
Should fix comiler timeouts that are occuring when we try to build anything since sccache isn't talking with s3 properly.

issue:
```
    sccache: error: Timed out waiting for server startup. Maybe the remote service is unreachable?
    Run with SCCACHE_LOG=debug SCCACHE_NO_DAEMON=1 to get more information
    ninja: build stopped: subcommand failed.
 ```

### CODE REVIEW

File: conda/recipes/kvikio/meta.yaml [hunk 1]
```diff
@@ -17,7 +17,21 @@ build:
...
+    - AWS_SESSION_TOKEN
+    - CMAKE_C_COMPILER_LAUNCHER
+    - CMAKE_CUDA_COMPILER_LAUNCHER
+    - CMAKE_CXX_COMPILER_LAUNCHER
+    - CMAKE_GENERATOR
     - PARALLEL_LEVEL
+    - SCCACHE_BUCKET
+    - SCCACHE_IDLE_TIMEOUT
+    - SCCACHE_REGION
+    - SCCACHE_S3_KEY_PREFIX=libkvikio-aarch64 # [aarch64]
```
Comment (bdice) [inline conda/recipes/kvikio/meta.yaml:31]:
```suggestion
    - SCCACHE_S3_KEY_PREFIX=kvikio-aarch64 # [aarch64]
```

File: conda/recipes/kvikio/meta.yaml [hunk 2]
```diff
@@ -17,7 +17,21 @@ build:
...
+    - SCCACHE_S3_KEY_PREFIX=libkvikio-linux64 # [linux64]
```
Comment (bdice) [inline conda/recipes/kvikio/meta.yaml:32]:
```suggestion
    - SCCACHE_S3_KEY_PREFIX=kvikio-linux64 # [linux64]
```

**Review by bdice — APPROVED** [review]
Thanks @robertmaynard. I thought I checked for this earlier but must have overlooked it! 😅

**Review by madsbk — APPROVED** [review]
Thanks @robertmaynard !

**Comment by jakirkham** [comment]
Thanks all! 🙏

**Pull request merged**